README.md

```
@@ -111,6 +110,8 @@ Secret names:
   - Google: `GoogleClientId`, `GoogleClientSecret`
   - Microsoft: `MicrosoftClientId`, `MicrosoftClientSecret`
   - Email OTP and Magic Link: `SmtpHost`, `SmtpPort`, `SmtpUser`, `SmtpPass`, `SmtpFrom`
+  - BetterAuth: `AuthSecret` (https://www.better-auth.com/docs/reference/options#secret))
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)